ui/package.json

```diff
@@ -109,6 +111,7 @@
     "cross-env": "^6.0.3",
     "dotenv-cli": "^2.0.1",
     "eslint": "^5.6.1",
+    "eslint-plugin-babel": "^5.3.0",
```

Comment:
Why are we adding this dependency?

Reply:
It might have been added by my local dev environment

Reply:
I'm not sure we need this dependency.

Reply:
I will remove it once the other dependency queries are resolved 😄 , Its just a dependency for `local development environment`.
Also I think should we include a section for `preferable development environment` for **TaskCluster**, Like using the react debugger and setting up `React-EsLint`.

Reply:
@helfi92 Should I remove this too ? As it will be used to solve local lint errors ?

Reply:
OK, I think we can remove it now.